cooking_recipe.cmake

```diff
@@ -194,6 +194,11 @@ cooking_ingredient (GnuTLS
     BUILD_COMMAND <DISABLE>
     INSTALL_COMMAND ${make_command} install)
 
+cooking_ingredient (zlib
+  EXTERNAL_PROJECT_ARGS
+    URL https://github.com/madler/zlib/releases/download/v1.3.1/zlib-1.3.1.tar.gz
+    URL_MD5 9855b6d802d7fe5b7bd5b196a2271655)
```

Comment:
i dropped zlib in 03cf327720055a7d325a891d7bdc28ac7c060978, which introduced this regression. as a fix, you might want to revert it. and mention the offending commit in the commit message. and if you think it'd be nice to bump up the version. i think it's fine to implement it in the same commit. but please explain the reason why 1.3.1 is picked.

Reply:
Thanks for the review. I changed the version back. And mentionded the offending commit in the PR's description.

Would you please have a look again?